DNN Platform/Library/Services/Installer/Installers/ScriptInstaller.cs

```diff
@@ -24,7 +24,8 @@ public class ScriptInstaller : FileInstaller
         private readonly SortedList<Version, InstallFile> _installScripts = new SortedList<Version, InstallFile>();
         private readonly SortedList<Version, InstallFile> _unInstallScripts = new SortedList<Version, InstallFile>();
         private InstallFile _installScript;
-        private InstallFile _upgradeScript;
+        private InstallFile _upgradeScript;
+        private readonly List<InstallFile> _schemaScripts = new List<InstallFile>();
```

Comment:
GIven that this is a non-sorted generic list, the order of execution would match the order if was input in the manifest.  Not saying that is wrong, but given the others are sorted by a systematic process should we sort here as well?

Reply:
I think the default "sort" of source order is the least surprising, unless we require some new attribute to indicate ordering.

Reply:
I'm ok with that as well, just calling out the difference vs the PR documentation which noted A-Z sorting

Reply:
Yeah, what @bdukes said. I have thought about adding ordering, but it would probably needlessly complicate matters. If a developer is particular about order, then they'll need to manage their manifest.